src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/helpers/TseHelper.java

```diff
@@ -293,4 +295,12 @@ private static List<GenericTypeItem<DocumentType>> getUploadedDocList(CaseData c
     private static String hasSupportingDocs(List<GenericTypeItem<DocumentType>> supportDocList) {
         return (supportDocList != null && !supportDocList.isEmpty())  ? "Yes" : "No";
     }
+
+    /**
+     * Gives current datetime in string format.
+     * @return current datetime in "yyyy-MM-dd'T'HH:mm:ss.SSS" format
+     */
+    public static String getCurrentDateTime() {
```

Comment:
Helper.getCurrentDateTime() can be used instead of recreating the method

Reply:
Done